server/app/api.py

```diff
@@ -1107,13 +1107,23 @@ def submit(self, user, assignment, messages, submit, submitter=None):
         due = valid_assignment.due_date
         late_flag = valid_assignment.lock_date and \
                     datetime.datetime.now() >= valid_assignment.lock_date
+        revision = valid_assignment.revision
 
         if submit and late_flag:
-            # Late submission. Do not allow them to submit
-            logging.info('Rejecting Late Submission', submitter)
-            return (403, 'late', {
-                'late': True,
-                })
+            if revision:
+                # In the revision period. Ensure that user has a previously graded submission.
+                fs = user.get_final_submission(valid_assignment)
+                if fs is None or fs.submission.get().score == []:
```

Comment:
Should we log this case as well with `logging.info` so that we can see how many submission failures are happening?